File: 005.js
var _ = require('underscore');

function isDivisible(x) {
	for(var i = 1; i <= 20; i++) {
		if(x % i != 0)
			return false;
	}
	return true;
}

var j = 20;
while(true) {
	if(isDivisible(j)) {
		console.log(j);
		return;
	}
	j++;
}

File: 005.py
def isDivisible(x):
	for i in range(2, 20):
		if x % i != 0:
			return False
	return True

number = 0
inc = 20

while(True):
	number += inc
	if isDivisible(number):
		print "Success!"
		print number
		break
